scripts/README.md

```
@@ -0,0 +1,316 @@
+   ：Set the following required variables:
+   
+   - `PR_NUMBER`: The pull request number for which metrics need to be fetched.
+   - `OWNER`: The owner of the repository (e.g., `my-org`).
+   - `REPO`: The name of the repository (e.g., `my-repo`).
+   - `GITHUB_TOKEN`: The token for authenticating with the Github API.
+   
+   - Optional variables (with default values):
+     - `DEFAULT_CODE_LEVEL`: Default code complexity level (`1`).
+     - `DEFAULT_NOT_ANALYSE_COMPLEXITY`: Default complexity for non-analyzed files (`1`).
+     - `DEFAULT_CPP_ENDS_WITH_FILE`: File extensions to analyze (`.cpp, .h, .hpp, .m, .mm, .cc` ）for C++
+     - `DEFAULT_PYTHON_ENDS_WITH_FILE `：File extensions to analyze (`.py` ）for Python
```

Markdownlint (0.37.0)

Spaces inside code span elements

(MD038, no-space-in-code)